README.md

```diff
@@ -177,7 +177,7 @@ var router = new Grapnel();
 router.get(expression, function(req, event){
     // GET http://mysite.com/page#food/tacos/good
     console.log('I think tacos are %s.', req.params[0]);
-    // => "He thinks tacos are good."
+    // => "I thinks tacos are good."
```

Comment:
The reason this comment is here and worded like this is because the machine has become self aware and is tired of me referencing tacos.